helm/botkube/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "botkube.fullname" . }}
+  labels:
```

Comment:
@Surbhidongaonkar We should take these labels from `Values.serviceMonitor.labels`

Reply:
@PrasadG193 small correction here: we should append the labels from `Values.serviceMonitor.labels` to these labels